### ISSUE TITLE
Add documentation for platformtest

### ISSUE THREAD

**Issue by sa7mon** [zrepl/zrepl#478]
Apologies if there already is documentation somewhere, but I wasn't able to find any for the `platformtest` tool anywhere. I noticed that my version of `zrepl` was out-of-date today so I went to grab the link for the 0.4.0 and saw in the instructions that I should run `platformtest` first. 

I downloaded and ran the binary only to be drowned in logs flying by with no idea of what it was doing. Eventually, it hung on the line: 
```log
2021-04-26T13:53:12-05:00 [DEBG][zfs.cmd][Uv38$g+Vq$g+Vq.FWqN]: start waiting cmd="zfs send zreplplatformtest/rootds/send er@a snapshot"
```
so I cancelled it and then had to manually delete the `zreplplatformtest` pool.

**Comment by problame** [comment]
Sorry for getting back to this with such a delay.

> Apologies if there already is documentation somewhere, but I wasn't able to find any for the platformtest tool anywhere. I noticed that my version of zrepl was out-of-date today so I went to grab the link for the 0.4.0 and saw in the instructions that I should run platformtest first.

:pray: 

> I downloaded and ran the binary only to be drowned in logs flying by with no idea of what it was doing. Eventually, it hung on the line

Curious, I haven't experienced that problem before.

- Are you using zrepl 0.4 in production?
- Do you encounter any related problems?
- What platform (OS, kernel version, ZFS version) are you running on?

**Comment by sa7mon** [comment]
> Are you using zrepl 0.4 in production?

It's just a home server

> Do you encounter any related problems?

I don't believe so

> What platform (OS, kernel version, ZFS version) are you running on?

OS: TrueNAS 12.2 (FreeBSD 12.2-RELEASE-p2)
ZFS version: zfs-2.0.0-1

**Comment by problame** [comment]
> It's just a home server

Ok, let me rephrase that: are you using `zrepl 0.4` successfully, despite the `platformtest` failing?

**Comment by sa7mon** [comment]
Yes, the sync jobs seem to be working fine.
